py/openage/convert/mediafile.py

```diff
@@ -77,6 +136,10 @@ def media_convert(args):
         args.extract.append('*:*.*')
 
     extraction_rules = [ExtractionRule(e) for e in args.extract]
+    if args.jobs == 0:
+        args.jobs = os.cpu_count()
```

Comment:
If there was a way to retrieve thread count from Pool, I'd love to do that. That's not possible and I want to let user know we're using X threads for conversion, so I decided to use `threads` for that.